roles/openshift_repos/tasks/main.yaml

```diff
@@ -37,12 +37,12 @@
   - "fedora-openshift-enterprise/repos/*"
   - "fedora-origin/repos/*"
   - "online/repos/*"
-  - "openshift-enterprise/repos/*"
   - "origin/repos/*"
   - "removed/repos/*"
   when: not openshift.common.is_containerized | bool
         and not (item | search("/files/" ~ openshift_deployment_type ~ "/repos"))
         and (ansible_os_family == "RedHat" and ansible_distribution != "Fedora")
+        and not openshift_deployment_type == 'openshift-enterprise'
   notify: refresh cache
 
 - name: Remove any yum repo files for other deployment types Fedora
```

Comment:
it looks this second loop isn't necessary, since fedora is included above.

Another option that we have is removing the use of with_fileglob and just hardcoding the paths to the files on disk as well.